src/BlockTypes/AbstractBlock.php

```diff
@@ -180,7 +180,7 @@ protected function register_chunk_translations( $chunks ) {
 	protected function get_chunks_paths( $chunks_folder ) {
 		$build_path = \Automattic\WooCommerce\Blocks\Package::get_path() . 'build/';
 		$blocks     = [];
-		if ( ! is_dir( $build_path . $chunks_folder ) ) {
+		if ( ! is_dir( $build_path ) || ! is_dir( $build_path . $chunks_folder ) ) {
```

Comment:
Aren't we sure that build_path is a directory?

Reply:
This is an outdated fix I submitted in #6464. I synced with `trunk`.